### ISSUE TITLE
No notification of ExifTool releases

### ISSUE THREAD

**Issue by boardhead** [exiftool/exiftool#13]
By the way, do you know why I do not get any notifications for exiftool releases despite watching the github project for "releases only"? I do that for dozens of projects and have never missed a notification. For exiftool I do not get any. Are you maybe suppressing notifications when adding new releases? (I doubt that, but I want to be sure...)

_Originally posted by @bersbersbers in https://github.com/exiftool/exiftool/issues/9#issuecomment-468392047_

**Comment by bersbersbers** [comment]
From your comment in #9:

1. I need to figure out the exact syntax to issue the POST command for exiftool.

My following comment in #9 has two examples using `curl`.

2. If I do this, will the version show up twice on the releases page since the tag already appears there?

I know too little about the mechanics of git tags to say anything definitive about that. But isn't a tag simply a name for a specific commit? Well, releases, too, are tags, so I expect GitHub to only show one of them if they point to the same commit. After all, there must be a way as seen in https://github.com/rg3/youtube-dl/releases - there is a tag for each release, and only the release is shown on the release page in that case. Maybe a version of this answer is to create the tag implicitly by creating the release, but I am unsure how that plays out with your potential non-GitHub git workflow.

**Comment by boardhead** [comment]
OK, thanks.  I'll try to do this for the next release (if I have time) to see how it goes.

**Comment by da2x** [comment]
You can subscribe to the appcast feed: https://github.com/exiftool/exiftool/releases.atom to be notified of all releases.

**Comment by bersbersbers** [comment]
Sure! Also, one can use https://gitpunch.com (which is what I do currently).

**Comment by kmilos** [comment]
Indeed, tagging is not enough, one has to actually create a "release" (as part of which tagging can be done as well): https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

**Comment by boardhead** [comment]
I won't be implementing this unless I can find a way to do it via a command line on a remote computer.  Otherwise it makes my release process unnecessarily more cumbersome.

**Comment by bersbersbers** [comment]
> I won't be implementing this unless I can find a way to do it via a command line on a remote computer.  

I thought you had found one in https://github.com/exiftool/exiftool/issues/9#issuecomment-468649933. 

Also, there is https://docs.github.com/en/rest/releases

**Comment by boardhead** [comment]
Right, thanks.  I need to find the time to figure out the exact syntax.

- Phil

**Comment by boardhead** [comment]
I was playing around with creating a 12.70 release to see how it worked, but I still don't know how to make it part of my automated release script.

There are mailing lists for both production and developement releases.  Email me if you want to be added (philharvey66@gmail.com).

- Phil

**Comment by boardhead** [comment]
I'm going to mark this as closed and not planned unless someone wants to step up and show me how to do this from the command line.  My time is better spent focusing on metadata issues.

- Phil

**Comment by bersbersbers** [comment]
https://github.com/settings/personal-access-tokens/new. Choose "Expiration", "No Expiration".

**Comment by boardhead** [comment]
Thanks for the help, but this isn't worth the trouble.

1. I've had to download and install 300+ MB worth of software to get "gh" running.
2. I wasted an hour trying to figure out how to generate a token that doesn't expire because I needed to change the exiftool organization settings to allow this.
3. After finally having everything in place to try this, I get the message:

A new release of gh is available: 2.27.0 → 2.69.0
To upgrade, run: brew upgrade gh
https://github.com/cli/cli/releases/tag/v2.69.0

Fuck me.  I used homebrew to install gh and apparently it didn't install the latest version.  But regardless, this highlights a single-point failure that will continue to waste my time in the future, and my release script will fail every time a new version of gh is issued.  This is not acceptable.  I'm throwing in the towel.

If anyone wants release notifications, send me an email and I'll add you to the mailing list.

- Phil

**Comment by woss** [comment]
You can subscribe to the official RSS or follow notifications of https://github.com/marketplace/actions/exiftool-action this. I have created an automated way to pick up the latest RSS version from the exiftool website, run the actions on github, and make the github release.

@boardhead, I have also forked this repo and added actions that help you do this with minimal effort. https://github.com/exiftool/exiftool/compare/master...woss:fork-exiftool:master 
Additionally, I believe I asked that on the forum, and it wasn't received with much enthusiasm.

**Comment by boardhead** [comment]
BTW, the recommended (and probably easiest and fastest) way to pick up the latest version from the ExifTool web site is to read https://exiftool.org/ver.txt

- Phil
